src/extension/app/components/action-bar/activity-action/activity-action.css.js

```diff
@@ -54,6 +55,10 @@ export const style = css`
     flex-grow: 1;
   }
 
+  .toast-container .message > svg {
```

Comment:
with the suggested change long messages looks as below 

<img width="885" alt="image" src="https://github.com/user-attachments/assets/3a653afa-e830-402a-9c58-633d82fa367b" />

Reply:
with the above change the toast container shows as 
<img width="883" alt="image" src="https://github.com/user-attachments/assets/cf8a131a-63ea-44b0-acb0-f77b6b747e57" />